examples/hubert/train.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+"""Train the HuBERTPretrainModel by using labels generated by KMeans clustering.
+Example:
+python train.py --root-path ./exp/data/mfcc/ --feature-type mfcc --num-classes 100
+"""
+
+import logging
+import pathlib
+from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser, RawDescriptionHelpFormatter
+from typing import Optional, Tuple
+
+from lightning import HuBERTPreTrainModule
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import ModelCheckpoint
+
+
+logger = logging.getLogger(__name__)
+
+
+class _Formatter(ArgumentDefaultsHelpFormatter, RawDescriptionHelpFormatter):
+    # https://stackoverflow.com/a/18462760
```

Comment:
maybe a little more here so that readers can quickly get a sense of the context before clicking through for more details